src/styles/index.scss

```diff
@@ -1,3 +1,137 @@
+$grey-color: #616070;
+$blue-color: #00acdc;
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+}
+
 body {
   margin: 0;
 }
+
+html {
+  font-family: Roboto, Manrope, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+}
+
+.product {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 32px 16px 16px;
+  width: 200px;
+  border-radius: 5px;
+  border: 1px solid #F3F3F3;
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+  }
+
+  &__title {
+    margin-bottom: 4px;
+    color: #060B35;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+  }
+
+  &__code {
+    width: 100%;
+    margin-bottom: 16px;
+    color: $grey-color;
+    font-size: 10px;
+    line-height: 14px;
+  }
+
+  &__price {
+    width: 100%;
+    margin-bottom: 16px;
+    display: flex;
+    justify-content: space-between;
+
+    &-text {
+      color: $grey-color;
+      font-size: 12px;
+      line-height: 18px;
+    }
+    &-values {
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+    }
+  }
+
+  &__reviews {
+    margin-bottom: 24px;
+  }
+
+  &__button {
+    width: 100%;
+    display: flex;
+    align-items: center;
+    justify-content: space-around;
+    border-radius: 5px;
+    border: 1px solid transparent;
+    background-color: $blue-color;
+    height: 40px;
+    color: #fff;
```

Comment:
not fixed 
![image](https://github.com/mate-academy/layout_product-cards/assets/70009205/c284082e-2537-43af-898c-b1270b40fe57)